pkgs/applications/networking/cluster/fn-cli/default.nix

```
@@ -0,0 +1,39 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
```

```suggestion
{ pkgs ? import <nixpkgs> {} }:

```

please do proper inputs!